app/containers/Registration/InputHook.js

```diff
@@ -0,0 +1,18 @@
+import {useState} from 'react';
+// Custom hook to handle form input changes
+export const useInput = (initialValue, onChange) => {
```

Comment:
This InputHook will need to be merged with the one in the Industry directory, and moved to the components directory (this is not a container)

Reply:
Yes, it needs to be in some common place. Me and Helen thought she can refactor after, but I can put it in components right away.

Reply:
Moved InputHook under components folder